Holodeck/Command.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from .Agents import *
+
+
+class Commands(object):
```

Comment:
"Commands" name isn't very informative. Please can you give it a more informative name like CommandGroup or something similar?
Also, please make the filename command.py to fit in with new naming conventions we will be moving to.